src/main/webapp/app/graph/graph.component.ts

```diff
@@ -25,60 +23,46 @@ import { RecommenderType } from '../../model/recommender-type';
 })
 /** The angular component that contains the graph and associated logic */
 export class GraphComponent implements OnInit {
-  public title = 'IAM Bindings';
-  public options: google.visualization.LineChartOptions = {
-    animation: {
-      duration: 250,
-      easing: 'ease-in-out',
-      startup: true
-    },
-    legend: { position: 'none' },
-    height: 700,
-    width: 1000,
-  }
-  public graphData;
-  public columns = ['Time', 'number', { type: 'string', role: 'tooltip' }];
+  /** The projects to display on the graph */
+  @Input()
+  public projects: Project[];
+
+  public options: google.visualization.LineChartOptions;
+  public graphData: any[];
```

Comment:
Honestly I'm not sure how. The graphData takes the form of of a bunch of rows in the following form:
```JavaScript
[time: Date, data1: number, data1-tooltip: string, data1-style: string, data2: number, data2-tooltip: string, ...]
```
Where `time` is the only field in that row that is never undefined. The other columns have to be there (undefined or otherwise) or else GCharts won't work at all